tests/message/message_renames.proto

```diff
@@ -32,13 +32,13 @@ message OuterMessageWithRenamedInnerMessage {
 }
 
 message MessageWithRenamedField {
```

Comment:
Can you add new tests with optional fields, rather than modify existing tests?

Probably should also test optional strings, bools too.